assets/stylesheets/sections/gutenberg-editor.scss

```diff
@@ -7,4 +7,39 @@
 @import '../shared/mixins/placeholder';
 @import '../shared/typography';
 
+// Editor package styles
+@import '../../../node_modules/@wordpress/editor/build-style/style';
+
+// Calypso specific Gutenberg editor styles
 @import 'gutenberg/editor/style';
+
+// Core blocks styles
+@import 'gutenberg/editor/core-blocks/paragraph/style';
+@import 'gutenberg/editor/core-blocks/heading/editor';
+
+// Edit-post components styles
+@import 'gutenberg/editor/edit-post/assets/stylesheets/colors';
```

Comment:
SCSS variables and helpers from the `assets` folder could probably be published to npm as its own package. What do you think?

Reply:
It would probably help us in the short term, but given that we are supposed to re-implement Calypso specific versions of edit post components, I'm guessing we won't be reusing them in the long run.

Reply:
not worth the effort then

Reply:
Depends on if core wants to eventually publish `edit-post` we'll untangle this in a bit otherwise in follow up PRs.